README.md

```diff
@@ -343,6 +343,19 @@ https://github.com/phbradley/tcr-dist
 
 > Software tools for the analysis of epitope-specific T cell receptor (TCR) repertoires
 
+Dash P, Fiore-Gartland A, Hertz T et al. [Quantifiable predictive features define epitope-specific T cell receptor repertoires.](https://pubmed.ncbi.nlm.nih.gov/28636592/) Nature. 2017;547(7661): 89-93.
+
+### tcrdist3
+
+Python scripts
```

Comment:
Could we change this to say "Python package" instead of "Python scripts"?

It looks like tcrdist3 is installable with pip like other python packages, whereas tcr-dist is not.